parts/windows/windowscontainerdfunc.ps1

```diff
@@ -93,9 +111,6 @@ function Enable-Logging {
     # !ContainerPlatformPersistent profile is made to work with long term and boot tracing
     & $diag -Start -ProfilePath "$global:ContainerdInstallLocation\ContainerPlatform.wprp!ContainerPlatformPersistent" -TempPath $logs
   }
-  else {

```

Comment:
why removing else?

Reply:
We can keep this. Let me bring this back.